db/seed/index.ts

```diff
@@ -25,6 +26,7 @@ async function main() {
       await coordinatorSeed(),
       await beneficiarySeed(),
       await campaignSeed(),
+      await bankAccountsSeed(),
```

Comment:
Let's make this method singular just like the other sibling functions
```suggestion
      await bankAccountSeed(),
```